containers.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: 2019-12-01
+location: uksouth
+name: lgDrupalContainerGroup
+properties:
+  containers:
+  - name: nginx-drupal
+    properties:
+      image: microsites-nginx-drupal:latest
+      resources:
+        requests:
+          cpu: 1
+          memoryInGb: 1.5
+      ports:
+        - port: 443
+  - name: drupal-fpm
+    properties:
+      image: microsites-drupal-fpm:latest
+      resources:
+        requests:
+          cpu: 1
+          memoryInGb: 1.5
+      environmentVariables:
+      - name: MYSQL_HOST
+        value: mariadb-ecc-gov-dev-svr.mariadb.database.azure.com
+      - name: MYSQL_USER
+        value: mariadb-root
+      - name: MYSQL_PASSWORD
+        secureValue: <PASSWORD>
+      - name: MYSQL_DATABASE
+        value: drupal
+      volumeMounts:
+      - mountPath: /drupal/web/sites/default/files
+        name: filesharevol
+      - mountPath: /drupal/data/default/private
+        name: privsharevol
+  osType: Linux
+  ipAddress:
+    type: Public
+    ports:
+    - protocol: tcp
+      port: 443
+    dnsNameLabel: ecc-microsites-drupal-dev
+  imageRegistryCredentials:
+  - server: eccgovreg.azurecr.io
+    username: eccmicrositesreg
+    password: <PASSWORD>
```

Comment:
@MatthewWilkes - Over in the [gov repo](https://github.com/essexcountycouncil/essex-gov-uk-drupal/blob/develop/containers.yaml#L46) the closing `>` is missing, which also made me wonder if this file is in use any longer.

Reply:
This isn't in use anymore, you can remove it.